src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "resources": {
+    "includes": [
```

Comment:
Can you describe the purpose of these exclusions? GraphQL Java has no direct relationship with `graphiql` for example.

Reply:
Hi! That's a good point. Feel free to not have those if you don't want. Somebody will need to provide those. Hopefully Spring Boot will do that one day soon...